src/core/Select/Select.tsx

```diff
@@ -51,7 +51,11 @@ export type SelectOption<T> = {
    * Item is disabled.
    */
   disabled?: boolean;
-};
+  /**
+   * Any other props.
+   */
+  [key: string]: unknown;
+} & CommonProps;
```

Comment:
This should also be added to `MenuItemProps` then

Reply:
only there or keep both?

Reply:
might not work if it's only there